<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1466</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QSplitter" name="topBottomSplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="topSplitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="codeDomainLayoutWidget">
        <layout class="QVBoxLayout" name="codeDomainParent">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Code</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QGroupBox" name="codeBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>Top Offenders</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QSplitter" name="varCodeSplitter">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <widget class="QWidget" name="codeVizLayoutWidget">
                <layout class="QVBoxLayout" name="codeVizLayout">
                 <item>
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>Source Lines</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="varVizLayoutWidget">
                <layout class="QVBoxLayout" name="varVizLayout">
                 <item>
                  <widget class="QLabel" name="label_11">
                   <property name="text">
                    <string>Data Objects</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QSplitter" name="splitter_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QSplitter" name="splitter_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QWidget" name="verticalLayoutWidget">
              <layout class="QVBoxLayout" name="codeEditorLayout">
               <item>
                <widget class="QLabel" name="codeLabel">
                 <property name="text">
                  <string>Source.cpp</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTabWidget" name="centerTabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="memoryTab">
         <attribute name="title">
          <string>Memory Topology</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>65</height>
               </size>
              </property>
              <property name="title">
               <string>Visualization Layout</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QRadioButton" name="memTopoVizModeIcicle">
                 <property name="text">
                  <string>Icicle</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="memTopoVizModeSunburst">
                 <property name="text">
                  <string>Sunburst</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>65</height>
               </size>
              </property>
              <property name="title">
               <string>Color by:</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QRadioButton" name="memTopoColorByCycles">
                 <property name="text">
                  <string># of cycles</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="memTopoColorBySamples">
                 <property name="text">
                  <string># of samples</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="memoryLayout"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="rightPaneLayoutWidget">
        <layout class="QVBoxLayout" name="rightPane">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="console">
           <property name="title">
            <string>Console</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <layout class="QVBoxLayout" name="consoleLayout"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="selectionMode">
           <property name="title">
            <string>Selection / Visibility</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QPushButton" name="selectAll">
                <property name="text">
                 <string>Select All</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="deselectAll">
                <property name="text">
                 <string>Deselect All</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="selectAllVisible">
                <property name="text">
                 <string>Select Visible</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16</height>
                 </size>
                </property>
                <property name="text">
                 <string>Select Mode:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="selectModeXOR">
                <property name="text">
                 <string>New</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="selectModeOR">
                <property name="text">
                 <string>Append</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="selectModeAND">
                <property name="text">
                 <string>Filter</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QPushButton" name="showAll">
                <property name="text">
                 <string>Show All</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="showSelectedOnly">
                <property name="text">
                 <string>Show Selected</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="hideSelected">
                <property name="text">
                 <string>Hide Selected</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="parallelCoordinatesLayoutWidget">
       <layout class="QVBoxLayout" name="parallelCoordinatesParent">
        <item>
         <widget class="QLabel" name="parallelCoordinatesLabel">
          <property name="text">
           <string>Parallel Coordinates</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="parallelCoordinatesParent_1">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="parallelCoordinatesLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item>
             <widget class="QGroupBox" name="parallelCoordinatesOptions">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="title">
               <string>PCoords Options</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <widget class="QLabel" name="opacityLabel">
                 <property name="text">
                  <string>Opacity:</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="opacityLayout">
                 <item>
                  <layout class="QVBoxLayout" name="selLayout">
                   <property name="sizeConstraint">
                    <enum>QLayout::SetFixedSize</enum>
                   </property>
                   <item>
                    <widget class="QLabel" name="selLabel">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>Sel</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSlider" name="selOpacity">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="maximum">
                      <number>999</number>
                     </property>
                     <property name="value">
                      <number>400</number>
                     </property>
                     <property name="sliderPosition">
                      <number>400</number>
                     </property>
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="invertedAppearance">
                      <bool>true</bool>
                     </property>
                     <property name="invertedControls">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="unselLayout">
                   <property name="sizeConstraint">
                    <enum>QLayout::SetFixedSize</enum>
                   </property>
                   <item>
                    <widget class="QLabel" name="unselLabel">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>Unsel</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSlider" name="unselOpacity">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="maximum">
                      <number>999</number>
                     </property>
                     <property name="value">
                      <number>100</number>
                     </property>
                     <property name="sliderPosition">
                      <number>100</number>
                     </property>
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="invertedAppearance">
                      <bool>true</bool>
                     </property>
                     <property name="invertedControls">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QCheckBox" name="histogramBox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Histograms</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1466</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_Data"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionImport_Data">
   <property name="text">
    <string>Load Data</string>
   </property>
  </action>
  <action name="actionImport_Source">
   <property name="text">
    <string>Select Source Directory</string>
   </property>
  </action>
  <action name="actionImport_Memory_Topology">
   <property name="text">
    <string>Import Memory Topology</string>
   </property>
  </action>
  <action name="actionAdd_Dataset">
   <property name="text">
    <string>Add Dataset</string>
   </property>
  </action>
  <action name="actionAdd_Dataset_2">
   <property name="text">
    <string>Add Dataset</string>
   </property>
  </action>
  <action name="actionLoad_Example">
   <property name="text">
    <string>Load Example</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
